test/unstructured.go

```diff
@@ -61,7 +62,7 @@ func WaitForUnstructuredState(ctx *Context, schema schema.GroupVersionResource,
 		lastState *unstructured.Unstructured
 		err       error
 	)
-	waitErr := wait.PollImmediate(Interval, Timeout, func() (bool, error) {
+	waitErr := wait.PollImmediate(Interval, 10*time.Minute, func() (bool, error) {
```

Comment:
Could you leave a comment in the code that we need the longer timeout for SeerviceMesh control plane?

Reply:
I'd prefer the comment in the commit as I did. This function can be used for other resources, not only Service Mesh. And I don't know if anyone will ever wonder why this is 10 minutes. I mean, both 5 and 10 minutes are just fine. That comment might be unnecessary. WDYT?